### PR TITLE
[FrameworkBundle] Add `AbstractController::renderBlock()` and `renderBlockView()`

### LINKED ISSUE #18764: [FrameworkBundle] Add `AbstractController::renderBlock()` and `renderBl…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#51327
| PR author(s) | @nicolas-grekas
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/18764

### CODE REVIEW

File: templates.rst [hunk 1]
```diff
@@ -604,6 +604,51 @@ to define the template to render::
...
+            // template block, so you can use those contents later in a `Response` object
+            $contents = $this->renderBlockView('product/index.html.twig', 'price_block', [
+                // ...
+            ]);
+
+            return new Response($contents);
+        }
+    }
+
+This might come handy when dealing with :ref:`templates inheritance <template_inheritance-layouts>`
```
Comment (94noni) [inline templates.rst]: Is it template inheritance or template block ?
Difference between extends and block need to be explicit imho
Anyway, that was documented fast 🚀

Comment (alexandre-daubois) [reply templates.rst]: Reworded a bit to mention blocks (which are mentioned in the inheritance doc), does it seem better?

**Review by 94noni — APPROVED** [review]
To me it is better indeed :)

**Review by alamirault — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Alex, thanks a lot for documenting this new feature!

Your contribution was perfect, but while merging we added a link to the AbstractController description for those who might need it: https://github.com/symfony/symfony-docs/commit/ebeb5ecd41ade61e4969c5e2e1e730afa9c6bc64

**Pull request merged**